### PR TITLE
added handling of currency prefixed merging costs

### PR DESCRIPTION
**Pull request by lucatume**
Ticket: https://central.tri.be/issues/33491

This handles merging costs coming with currency. EB differs in return values...

### CODE REVIEW

**Comment by lucatume** [comment]
P.S.: I've run the tests and those are still green.

File: src/Tribe/Cost_Utils.php [hunk 1]
```diff
@@ -449,4 +450,12 @@ private function filter_numeric_values( $value ) {
 	private function restore_original_decimal_separator( $value ) {
 		return str_replace( '.', $this->_current_original_cost_separator, $value );
 	}
+
+	private function numerize_numbers( $value ) {
```
Comment (borkweb) [inline src/Tribe/Cost_Utils.php]: Can you add a docblock to this method to describe what it does?

Comment (lucatume) [inline src/Tribe/Cost_Utils.php]: Ok, will do.
For future reference: are `private` and `protected` methods too to be documented?

**Comment by borkweb** [comment]
:+1: Looks good!

**Pull request merged**